### ISSUE TITLE
不同平台训练的时候日志打印会不一样？

### ISSUE THREAD

**Issue by linkewei0580** [PaddlePaddle/PaddleOCR#5657]
其他配置都一样的，为啥WINDOWS下不会打印“l[2022/03/08 14:18:44] root INFO: epoch: [3/1200], iter: 2, lr: 0.000500, loss: 2.568993, loss_shrink_maps: 1.569904, ”

centos7   CPU
[root@filebackup PaddleOCR]# git branch
* release/2.4
[2022/03/08 14:12:02] root INFO: profiler_options : None
[2022/03/08 14:12:02] root INFO: train with paddle 2.2.2 and device CPUPlace
[2022/03/08 14:12:02] root INFO: Initialize indexs of datasets:['/home/PaddleOCR/train_data/det/train.txt']
[2022/03/08 14:12:02] root INFO: Initialize indexs of datasets:['/home/PaddleOCR/train_data/det/val.txt']
[2022/03/08 14:12:03] root INFO: load pretrain successful from ./pretrain_models/ch_ppocr_server_v2.0_det_train/best_accuracy
[2022/03/08 14:12:03] root INFO: train dataloader has 1 iters
[2022/03/08 14:12:03] root INFO: valid dataloader has 2 iters
[2022/03/08 14:12:03] root INFO: During the training process, after the 3000th iteration, an evaluation is run every 2000 iterations
[2022/03/08 14:12:03] root INFO: Initialize indexs of datasets:['/home/PaddleOCR/train_data/det/train.txt']
[2022/03/08 14:14:11] root INFO: epoch: [1/1200], iter: 0, lr: 0.000000, loss: 3.017450, loss_shrink_maps: 1.888532, loss_threshold_maps: 0.751774, loss_binary_maps: 0.377144, reader_cost: 0.82888 s, batch_cost: 64.00040 s, samples: 6, ips: 0.04687
[2022/03/08 14:14:12] root INFO: save model in ./output/ch_db_res18/latest
[2022/03/08 14:14:12] root INFO: Initialize indexs of datasets:['/home/PaddleOCR/train_data/det/train.txt']
[2022/03/08 14:16:26] root INFO: epoch: [2/1200], iter: 1, lr: 0.000250, loss: 2.752391, loss_shrink_maps: 1.710488, loss_threshold_maps: 0.700637, loss_binary_maps: 0.341266, reader_cost: 0.94188 s, batch_cost: 67.07004 s, samples: 6, ips: 0.04473
[2022/03/08 14:16:27] root INFO: save model in ./output/ch_db_res18/latest
[2022/03/08 14:16:27] root INFO: Initialize indexs of datasets:['/home/PaddleOCR/train_data/det/train.txt']
l[2022/03/08 14:18:44] root INFO: epoch: [3/1200], iter: 2, lr: 0.000500, loss: 2.568993, loss_shrink_maps: 1.569904, loss_threshold_maps: 0.685344, loss_binary_maps: 0.313745, reader_cost: 0.92959 s, batch_cost: 68.19736 s, samples: 6, ips: 0.04399
[2022/03/08 14:18:45] root INFO: save model in ./output/ch_db_res18/latest
[2022/03/08 14:18:45] root INFO: Initialize indexs of datasets:['/home/PaddleOCR/train_data/det/train.txt']

windows GPU
E:\PaddleOCR>git branch
* release/2.4
[2022/03/08 14:02:10] root INFO: profiler_options : None
[2022/03/08 14:02:10] root INFO: train with paddle 2.2.2 and device CUDAPlace(0)
[2022/03/08 14:02:10] root INFO: Initialize indexs of datasets:['E:\\PaddleOCR\\train_data\\det\\train.txt']
[2022/03/08 14:02:10] root INFO: Initialize indexs of datasets:['E:\\PaddleOCR\\train_data\\det\\val.txt']
W0308 14:02:10.582033  5124 device_context.cc:447] Please NOTE: device: 0, GPU Compute Capability: 8.6, Driver API Version: 11.6, Runtime API Version: 11.2
W0308 14:02:10.592003  5124 device_context.cc:465] device: 0, cuDNN Version: 8.1.
W0308 14:02:11.675161  5124 device_context.h:397] WARNING: device: 0. The installed Paddle is compiled with CUDNN 8.2, but CUDNN version in your machine is 8.1, which may cause serious incompatible bug. Please recompile or reinstall Paddle with compatible CUDNN version.
[2022/03/08 14:02:12] root INFO: load pretrain successful from ./pretrain_models/ch_ppocr_server_v2.0_det_train/best_accuracy
[2022/03/08 14:02:12] root INFO: train dataloader has 1 iters
[2022/03/08 14:02:12] root INFO: valid dataloader has 2 iters
[2022/03/08 14:02:12] root INFO: During the training process, after the 3000th iteration, an evaluation is run every 2000 iterations
[2022/03/08 14:02:12] root INFO: Initialize indexs of datasets:['E:\\PaddleOCR\\train_data\\det\\train.txt']
[2022/03/08 14:02:13] root INFO: save model in ./output/ch_db_res18/latest
[2022/03/08 14:02:13] root INFO: Initialize indexs of datasets:['E:\\PaddleOCR\\train_data\\det\\train.txt']
[2022/03/08 14:02:14] root INFO: save model in ./output/ch_db_res18/latest
[2022/03/08 14:02:14] root INFO: Initialize indexs of datasets:['E:\\PaddleOCR\\train_data\\det\\train.txt']
[2022/03/08 14:02:15] root INFO: save model in ./output/ch_db_res18/latest
[2022/03/08 14:02:15] root INFO: Initialize indexs of datasets:['E:\\PaddleOCR\\train_data\\det\\train.txt']

**Comment by tink2123** [comment]
训练样本只有一张图吗？ 可以试下增大训练集训练，例如8张图。

**Comment by linkewei0580** [comment]
已经找到问题，和#5710 是同一个问题
